src/Framework/Traits.cs

```diff
@@ -392,29 +392,6 @@ public SdkReferencePropertyExpansionMode? SdkReferencePropertyExpansion
             }
         }
 
-        /// <summary>
-        /// Allows displaying the deprecation warning for BinaryFormatter in your current environment.
-        /// </summary>
-        public bool EnableWarningOnCustomBuildEvent
-        {
-            get
-            {
-                var value = Environment.GetEnvironmentVariable("MSBUILDCUSTOMBUILDEVENTWARNING");
```

Comment:
We should update (or create dotnet/docs bug requesting the update) the https://learn.microsoft.com/en-us/dotnet/core/compatibility/sdk/8.0/custombuildeventargs - to remove the mention of the opt-out

Reply:
Good point. #11354